edxval/tests/test_api.py

```diff
@@ -1333,37 +1380,158 @@ def test_invalid_course_id(self):
         xml = self.make_import_xml(video_dict=constants.VIDEO_DICT_FISH)
         self.assert_invalid_import(xml, "x" * 300)
 
+    # FIXME: EDUCATOR-2403
     def test_external_no_video_transcript(self):
         """
         Verify that transcript import for external video working as expected when there is no transcript.
         """
-        api.import_from_xml(etree.fromstring('<video_asset/>'), '')
+        api.import_from_xml(
+            etree.fromstring('<video_asset/>'),
+            '',
+            self.file_system,
+            constants.EXPORT_IMPORT_STATIC_DIR
+        )
         self.assertEqual(
             VideoTranscript.objects.count(),
             0
         )
 
+    def test_import_transcript_attached_existing_video(self):
```

Comment:
we also need a test which verify the below
```markdown
1. we have video with `edx_video_id=12345` with some transcripts
3. do import with new transcripts
4. verify that video with `edx_video_id=12345` has both the old and new transcripts
```

Reply:
If we have video with some edx_video_id in DS then we don't do import for it (therefore transcript import is also not done)